crates/ruff/src/rules/flake8_bugbear/rules/abstract_base_class.rs

```diff
@@ -73,6 +75,26 @@ fn is_empty_body(body: &[Stmt]) -> bool {
     })
 }
 
+fn fix_abstractmethod_missing(checker: &Checker, stmt: &Located<StmtKind>) -> Result<Fix> {
+    let indent =
+        indentation(checker.locator, stmt).ok_or(anyhow!("Unable to detect indentation"))?;
```

Comment:
Nit: We could also use `unwrap_or_default` because `indentation` returning `None` means that the `stmt` is at the beginning of the line.

Reply:
Ooooh that makes a ton of sense, would explain the None. I guess that function only looks at the line of the Located, so it'd make sense if it failed for unindented lines.

I think end result will be the same regardless (this should never hit an unindented class method, so it should never be None), but unwrap_or_default feels like it'll read better anyways. Let me make that change real fast.

Reply:
Updated, looks good!